pkg/kando/location.go

```diff
@@ -20,14 +20,22 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	"github.com/kanisterio/kanister/pkg/datamover"
 	"github.com/kanisterio/kanister/pkg/param"
 )
 
 const (
-	pathFlagName    = "path"
-	profileFlagName = "profile"
+	pathFlagName             = "path"
+	profileFlagName          = "profile"
+	repositoryServerFlagName = "repository-server"
 )
 
+type DataMoverArgs struct {
+	Type             string
```

Comment:
Let's declare and use DataMoverType as a new type. Something like this - https://github.com/kanisterio/kanister/blob/master/pkg/apis/cr/v1alpha1/types.go#L163

Reply:
Done !